### DISCUSSION TITLE
Search Feature

### DISCUSSION THREAD

**Discussion by bencrazy** [monicahq/monica#7181] [Ideas]
Just started using Monica and so far really like it. One thing I noticed is missing (or maybe it's just me that's missing it) is the ability to search fields, not only a contact name (for example search by company name, street name, title, etc).

Is this something that would be easy to implement and is there any interest in expanding the search feature?

Keep up the great work!

**Comment by fredsavoir** [comment]
Hi, I'm looking for the same feature, will it be in Version 5?
Thanks.
